include/cutlass/gemm/collective/xe_mma.hpp

```diff
@@ -261,7 +267,9 @@ struct CollectiveMma<
         prefetch(mainloop.gmem_tiled_copy_b, iter_b(_,_,_,k_tile + DispatchPolicy::Stages));  
       }
     }
-    cute::gemm(tiled_mma, accum, tCrA, tCrB, src_accum);
+    for (int i = 0; i < SG_K / SubgroupSize; i++) {
```

Comment:
Using the following configuration we are still getting the wrong result, even when using this PR.
```c++
using GmemTiledCopyA = XE_2D_U16x8x16_LD_N;
using GmemTiledCopyB = XE_2D_U16x16x16_LD_V;

using TileShape = Shape<_256, _256, _32>;

using TiledMma = TiledMMA<MMA_Atom<XE_8x16x16_F32BF16BF16F32_TT>,
       Layout<Shape<_4,_4,_2>>,
       Tile<_32,_64,_32>>;
```
by having a quick look into the code, it seems that the logic used in this for loop does not consider the   `Layout<Shape<_4,_4,_2>>` value when using the Atom `XE_8x16x16_F32BF16BF16F32_TT` to fulfil the computation of `Tile<_32,_64,_32>`. It seems that this for loop divide the K value with Subgroup-Size irrespective of K size in the thread group layout assuming it is always 1. So when the K size is the thread group layout is 2 (e.g. `Layout<Shape<_4,_4,_2>>`), it seems that we get a wrong result. Would you be able to investigate that?

Reply:
Thanks, your observation is right. Actually SG_K and BLK_K share the same parameter.
I am wondering whether  `Layout<Shape<_4,_4,_2>>,`  is  set for split-K algorithm ?

Reply:
When use `Layout<Shape<_4,_4,_2>>`, this means that only partial sum will be obtained ?

Reply:
In this PR's, our goal is implementing the CUTE APIs which used in existing examples, and make sure there is no regression. How about merge this PR and make sure the CI/CD can work first. (BTW, the layout cannot work before this PR)

Yes, there are some corner cases we don't implement yet including the layout you mentioned, how about to file a JIRA/task to track these requirements?

Reply:
Same here, generally this for loop should be handle inside the cut::gemm not here. Lets create a  ticket to address this separetly